uvahelpr_exp/uvahelpr_exp/views.py

```diff
@@ -15,11 +15,23 @@ def get_all_jobs(request):
     return HttpResponse(result)
 
 def job_summary(request, id):
-    req = urllib.request.Request('http://' + request.META['SERVER_NAME'] + ':8001/api/v1/jobs/' + id + '/')
+    req = urllib.request.Request('http://models-api:8000/api/v1/jobs/' + id + '/')
     resp_json = urllib.request.urlopen(req).read().decode('utf-8')
     resp_dict = json.loads(resp_json)
     job = resp_dict['result']
-    newjob = {'id': job['id'], 'title': job['title'], 'description': job['description'], 'time_required': job['time_required'],
-              'event_time': job['event_time'], 'skills_required': job['skills_required'], 'compensation': job['compensation'], 'location':job['location']}
+    user_id = str(job['requester_id'])
+
+    req_user = urllib.request.Request('http://models-api:8000/api/v1/users/' + user_id + '/')
+    user_json = urllib.request.urlopen(req_user).read().decode('utf-8')
+    user_dict = json.loads(user_json)
+    user_info = user_dict['result']
+    firstname = user_info["first_name"]
+    lastname = user_info['last_name']
+    name = firstname + ' ' + lastname
+
+    newjob = {x: job[x] for x in ['id', 'title', 'description', 'time_required', 'event_time', 'skills_required', 'compensation', 'location']}
+    newjob['requester'] = name
+
     result = json.dumps(newjob)
-    return HttpResponse(result)
\ No newline at end of file
+    return HttpResponse(result)
```

Comment:
Like dad said, this should be using jsonresponse instead of httpresponse. Remove the json.dumps line from before that since we don't need it to be a string for jsonresponse
https://docs.djangoproject.com/en/1.8/ref/request-response/#jsonresponse-objects